tests/test_podcast_search.py

```
@@ -0,0 +1,497 @@
+        with patch.object(
+            time,
+            'time',
+            return_value=time.time() - _CITATION_TTL_SECONDS - 100
+        ):
+            set_podcast_citations(session_id, citations)
+
+        # Next set_podcast_citations should trigger cleanup
+        set_podcast_citations("another-session", [])
+
+        # Original session should be cleaned up
+        result = get_podcast_citations(session_id)
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)